README.md

```diff
@@ -7,55 +7,65 @@ query (or query template) as a table.
 
 ## Configuration
 
-These are the configuration keys that need to be set in the plugin
-configuration, via the Linkurious Configuration.
-
-- `entityType` (**required**, `"edge"` or `"node"`): whether to display
-  nodes or edges in the table (e.g. `"node"`).
-- `itemType` (**required**, string): name of the node-category or
-  edge-type to display in the table (e.g. `"COMPANY"`).
-- `properties` (**optional**, array): property names to include in the
-  table (e.g. `["name", "age", "date_of_birth"]`). Defaults to all
-  properties of the configured `itemType`.
-- `basePath` (**optional**, string): URL path in which the plugin will
-  be deployed (e.g. `"my-plugin"`). Defaults to plugin name.
-
-Full configuration example:
-```json
-{
- "data-table": {
-    "basePath": "table",
-    "entityType": "node",
-    "itemType": "Company",
-    "properties": ["name", "address", "vat_number"]
- }
-}
-```
+This plugin supports the following configuration keys:
+
+| Key | Type | Description | Example |
+| :-- | :-- | :-- | :-- |
+| `basePath`    | string (**optional**)            | A base path on which the plugin will be mounted. Defaults to `"data-plugin"`. | `"table"` |
+| `debugPort`   | number (**optional**)            | A debug port on which to attach a debugger for the plugin NodeJS process. | `9230` |
```

Comment:
I would put this as last

Reply:
This was not originally included before this PR, and I was actually tempted to separate `basePath` and `debugPort` in another table labeled as common config keys for all plugins. Wdyt? separate table or just move below? @francescoinfante

Reply:
@renato-rg yes it would make sense this split to me (just one after the other)

Reply:
I split them into 2 tables, review it again